validator/tools/sanitize_dashboards.js

```diff
@@ -0,0 +1,42 @@
+const { DEFINITIONS_DIR, DASHBOARD_FILE_NAME_SUFFIX, FILE_ENCODING} = require('./props');
+const {readdir} = require('fs').promises;
+const fs = require('fs');
+const path = require('path');
+
+(async () => {
+    const folderDefinitions = await readdir(DEFINITIONS_DIR, {withFileTypes: true})
+    for (let folderDefinition of folderDefinitions){
+        if(folderDefinition.isDirectory()){
```

Comment:
worth raising errors on the negative of the folder and file branches?

Reply:
If we have an unexpected file here you mean? 🤔 there's another script that validates folders so here I added this just in case